src/style.css

```
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  
+
+  --page-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+  align-self: center;
+}
+
+.nav_list {
+  --offset-left: 20px;
+
+  
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  margin-left: calc(-1 * var(--offset-left));
+}
+
+.nav_item {
+  margin-left: var(--offset-left);
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+  white-space: nowrap;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav_link.is-active,
+.nav_link:hover {
+  color: var(--page-color);
+}
+
+.nav_link::after {
+  content: "";
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  background-color: var(--page-color);
+  border-radius: 2em;
+  opacity: 0;
+}
+
+.nav_link.is-active::after {
+  opacity: 1;
+}
```

No need for this in this task
```suggestion
.nav_link.is-active::after {
  opacity: 1;
}
```